examples/evaluation_and_profiling/swe_bench/src/nat_swe_bench/config.py

```
@@ -16,17 +16,28 @@
+from nat.data_models.component_ref import FunctionRef
+from nat.data_models.component_ref import LLMRef
 from nat.data_models.function import FunctionBaseConfig
 
 
 class SweBenchPredictorBaseConfig(TypedBaseModel, BaseModelRegistryTag):
     description: str = "Swe Bench Problem Solver"
 
 
+class SweBenchPredictorFullConfig(SweBenchPredictorBaseConfig, name="full"):
+    llm_name: LLMRef = "nim_llm"
+    tool_names: list[FunctionRef] = []
```

Ruff (0.14.11)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)